src/transaction.js

```diff
@@ -176,7 +176,7 @@ Transaction.prototype.addInput = function (hash, index, sequence, scriptSig) {
 }
 
 Transaction.prototype.addOutput = function (scriptPubKey, value) {
-  typeforce(types.tuple(types.Buffer, typeforce.anyOf(types.Satoshi, types.Buffer)), arguments)
```

Comment:
`anyOf` was added in `1.17.0` but we are on `1.13.2` on mobile so I'm using `oneOf` as it matches our semver.